shared/src/commands/validate/validate.ts

```diff
@@ -42,38 +43,34 @@ export function exitBasedOffOfValidationOutcome(validationOutcome: ValidationOut
     process.exit(0);
 }
 
-export function getFormattedOutput(
+export enum OutputFormat {
```

Comment:
enum's are generally avoided in Typescript - unless there's a reason I'm missing, I think we should do `type OutputFormat = 'junit' | 'json' | 'pretty'` instead. That also eliminates the need for a default branch in the switch statement below